scripts/copyComponentExperiencesYaml.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const path = require('path');
+const process = require('process');
+const { execSync } = require('child_process');
+
+const projectRoot = path.resolve(__dirname, '..');
+const outDir = path.join(projectRoot, 'out', 'src', 'resources');
+const sourceFile = path.join(projectRoot, 'resources', 'component-experiences.yaml');
+const destFile = path.join(outDir, 'component-experiences.yaml');
+
+fs.mkdirSync(outDir);
+
+if (process.platform === 'win32') {    
```

Comment:
I'd take advantage of Node's file I/O and do away with this conditional as well. I.e.:

```node
fs.copyFileSync(sourceFile, destFile);
```